storage/coin.go

```diff
@@ -14,7 +10,5 @@ type Coin struct {
 	IsNativeToken   bool           `json:"is_native_token"`
 	Logo            string         `json:"logo"`
 	PriceProviderID string         `json:"price_provider_id"`
-	Decimals        int32          `json:"decimals"` // Add the missing field
-	RawBalance      string         `json:"raw_balance"`
-	PriceRate       common.Decimal `json:"price_rate"`
+	Decimals        int32          `json:"decimals"`
```

Comment:
why we remove `raw_balance` and `price_rate`?  I thought we could keep it ,  so the coin's last balance and price rate will be cached in this way , user will be able to see it even they are offline

Reply:
We don't have a mechanism to synchronize those fields, so maintaining them would just create extra work.

If we agree to use react-query for data fetching, it will handle caching automatically, and we can easily enable persistent caching to support offline functionality.

Caching can be managed at a single react-query level, so we don't need to worry about implementing custom syncs for every data type.

Reply:
ok, we can deal this a bit later , and then decide which one is better